scripts/mysql_system_users.sql

```diff
@@ -78,7 +78,10 @@ REVOKE ALL PRIVILEGES, GRANT OPTION FROM 'mysql.pxc.internal.session'@localhost;
 
 -- Due to bugs with roles, we need to grant superuser access here
 GRANT ALL PRIVILEGES ON *.* TO 'mysql.pxc.internal.session'@localhost WITH GRANT OPTION;
```

Comment:
Here, we are granting all privileges to `mysql.pxc.internal.session`, is it still required to grant individual privileges separately to `mysql.pxc.internal.session`?

Reply:
As we discussed, no, it is not needed. Some historical mess. Removing.